### ISSUE TITLE
AttributeError: module 'nbconvert.preprocessors.execute' has no attribute 'CellExecutionError'

### ISSUE THREAD

**Issue by nilswagner** [spatialaudio/nbsphinx#466]
How can I resolve the following issue?

nbsphinx/src/nbsphinx.py", line 995, in parse
    except nbconvert.preprocessors.execute.CellExecutionError as e:
AttributeError: module 'nbconvert.preprocessors.execute' has no attribute 'CellExecutionError'

**Comment by mgeier** [comment]
Are you using a pre-release version of `nbconvert`?

See also https://github.com/jupyter/nbconvert/issues/1237.

**Comment by nilswagner** [comment]
>>> nbconvert.__version__
'6.0.0a3'

**Comment by mgeier** [comment]
Possible work-arounds:

* use `nbconvert < 6`
* apply this patch: https://github.com/mgeier/nbsphinx/commit/b90cfb5d2a8cabf70d5dd21d1c105e71a334dc06
* ask the `nbconvert` people to provide a smooth transition instead of a hard breaking change, see https://github.com/jupyter/nbconvert/issues/1237

I cannot properly fix this yet in `nbsphinx`, because the `nbconvert` people might still make further breaking changes before `nbconvert` version 6 is released, see https://github.com/jupyter/nbconvert/issues/1237.

As a temporary measure, I could restrict `nbconvert` to `<6` in the `nbsphinx` dependencies, but I'm not sure that's the right thing to do.
What do you think?

**Comment by mgeier** [comment]
This has been fixed in `nbconvert` in the meantime.

**Comment by HealthyPear** [comment]
I am getting this exact error but from a sphinx build

```
Sphinx parallel build error:
AttributeError: module 'nbconvert.preprocessors.execute' has no attribute 'CellExecutionError'
```

I am using 

```
nbconvert                 7.13.0             pyhd8ed1ab_0    conda-forge
nbconvert-core            7.13.0             pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.13.0             pyhd8ed1ab_0    conda-forge
nbsphinx                  0.9.3              pyhd8ed1ab_0    conda-forge
```

both seem to be at the latest release

Let me know if you'd prefer I open a new issue

**Comment by HealthyPear** [comment]
from a python prompt this works

`from nbconvert.preprocessors import CellExecutionError`

but this doesn't

`from nbconvert.preprocessors.execute import CellExecutionError`

as the error indeed reports

**Comment by HealthyPear** [comment]
see #772

**Comment by mgeier** [comment]
This seems to have been removed in https://github.com/jupyter/nbconvert/pull/2079, maybe unintentionally.

But since there has been a new `nbconvert` release since, I guess we should fix it. Thanks for providing the fix in #772!

**Comment by mgeier** [comment]
A new patch-release of `nbconvert` has just been released, which should fix this again.

I think we should still change the import on our side.
